### DISCUSSION TITLE
列特别多的情况加上自定义单元格，还有固定列，这种情况下，多选框选择会非常慢怎么办？

### DISCUSSION THREAD

**Discussion by jqhr** [ElemeFE/element#22538] [Q&A]
我的表格列比较多，50+列，而且列包含合并行还有和并列，加上带有固定列，选中非常慢，怎么办？
带上固定列，相当于渲染了两个单元格组件，我这里的单元格都是自定义的组件，怎么办？
有什么优化的建议？

**Comment by webvs2** [comment]
Please provide a simple example for review

**Comment by wangdaodao** [comment]
如果不考虑浏览器兼容，我觉得可以使用position: sticky去做性能优化，另外可以加上虚拟滚动来优化。
